doc/ug/getting_started_sw_testing.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Test Software
+---
+
+## Prerequisites
+
+_Make sure you followed the install instructions to [prepare the system]({{< relref "install_instructions#system-preparation" >}}) and to install the [software development tools]({{< relref "doc/ug/install_instructions#software-development" >}}) and [Verilator]({{< relref "install_instructions#verilator" >}})._
+
+## Overview
+
+The OpenTitan software tests fall into two main categories:
+
+* Unit tests
+* Smoke tests
```

Comment:
I'd make this category a bit wider: it's on-device tests, with smoke tests being a subset of them. We also have full applications that run on-device, e.g. tests that perform a full RSA or ECDSA sign/verify.